lib/slackistrano/tasks/slack.rake

```diff
@@ -42,6 +42,5 @@ end
 before 'deploy:starting',           'slack:deploy:starting'
 before 'deploy:updating',           'slack:deploy:updating'
 before 'deploy:reverting',          'slack:deploy:reverting'
-after  'deploy:finishing',          'slack:deploy:updated'
```

Comment:
I am not very well versed in Slackistrano, but it looks like this totally removes the final updated message. Don't we still want that message, just later?

Reply:
Yeah, I will introduce the replacement in our code.

Here:

https://github.com/LearnZillion/learnzillion/blob/01c764e0b457d45bc46892dc57b44bfd61b88d63/lib/capistrano/tasks/notify.rake#L60

Reply:
Basically, i allows us to re-introduce in LearnZillion code anyway we choose.

Reply:
Ah I see. Thank you!